assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -1094,6 +1094,20 @@ def validate_early_stop_settings(args: Namespace) -> None:
         )
 
 
+def validate_optimiser_name(args: Namespace) -> None:
+    """Validate optimiser name allowing only 'adamw_torch' and 'adafactor'."""
+    optim = str(args.optim).lower()
+    if optim not in ("adamw_torch", "adafactor"):
```

Comment:
can this be a static list? just in case tomorrow if we add another optimiser updating the list would be sufficient? accordingly update the error message to construct a string from this list instead of hardcoding in the log?